docs/reference/sql/endpoints/cli.asciidoc

```diff
@@ -47,3 +43,27 @@ James S.A. Corey |Leviathan Wakes     |561            |1306972800000
 --------------------------------------------------
 // TODO it'd be lovely to be able to assert that this is correct but
 // that is probably more work then it is worth right now.
+
+The jar containing the SQL CLI is a stand alone Java application and
+the scripts just launch it. You can move it around to other machines
+without having to install Elasticsearch on them. Without the already
+provided script files, you can use a command similar to the following
+to start the SQL CLI. A Java version not earlier than the one used to 
```

Comment:
Couldn't be compiled with Java 13 and run with Java 12 (for earler 7.x versions) ?